### PR TITLE
Laravel 6.0 Update

### PR DESCRIPTION
**Pull request by FaizAhmadSE**

### CODE REVIEW

File: .github/actions/common/setup [hunk 1]
```diff
@@ -8,9 +8,6 @@ git checkout -b $commit
 composer create-project --prefer-dist laravel/laravel test-app
 cd test-app
 
-# Scaffold Authentication
```
Comment (DCzajkowski) [inline .github/actions/common/setup:11]: Why did you remove this from GitHub actions? If you remove this there will be no auth scaffolding. This is what makes tests fail here: https://github.com/DCzajkowski/auth-tests/pull/28/checks?check_run_id=212056515

File: README.md [hunk 2]
```diff
@@ -1,43 +1,54 @@
...
 ## Versioning
+
 The version of this package reflects current major version of the Laravel framework. For example:
 If Laravel framework has version 5.6, version of this package compatible will be `5.6.*`.
 
 ## Installation
+
 ```bash
 composer require dczajkowski/auth-tests --dev
-php artisan make:auth # if not ran previously
```
Comment (DCzajkowski) [inline README.md:15]: I think this is important to let people know running `make:auth` is required for tests to pass.

File: composer.json [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
-                "DCzajkowski\\AuthTests\\AuthTestsServiceProvider"
-            ]
-        }
-    },
-    "autoload": {
-        "files": [
-            "src/AuthTestsServiceProvider.php",
-            "src/Console/Commands/AuthTestsMakeCommand.php"
-        ]
+  "name": "dczajkowski/auth-tests",
```
Comment (DCzajkowski) [inline composer.json:2]: I understand the desire to use 2 spaces in `.json` files, but [Laravel uses 4 spaces](https://github.com/laravel/laravel/blob/master/composer.json), so I would stick with that.

**Review by DCzajkowski — CHANGES_REQUESTED** [review]
Hello, thank you for the PR. I have done a simple pass reviewing the changes, but I didn't have time to try it out. Please fix mentioned issues and I will try it out in the upcoming days

File: README.md (hunk 2 above)
Comment (drbyte) [reply README.md:15]: The `make:auth` command was removed in Laravel 6.
Now the initial UI scaffolding can be done by installing `laravel/ui` package and running `php artisan ui bootstrap --auth` or php artisan ui vue --auth` and running `npm install && npm run dev`

Comment (DCzajkowski) [reply README.md:15]: Oh, I did not know that. Haven't had time to look at 6.0 release notes at all. Thanks for the heads up! I will look into that in the upcoming days.
